src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/Registry.cs

```diff
@@ -62,18 +62,17 @@ private static RegistryKey GetBaseKeyFromKeyName(string keyName, out string subK
                     string.Empty :
                     keyName.Substring(i + 1, keyName.Length - i - 1);
 
-                return baseKey;
+                return baseKey!;
             }
 
             throw new ArgumentException(SR.Format(SR.Arg_RegInvalidKeyName, nameof(keyName)), nameof(keyName));
         }
 
-        public static object GetValue(string keyName, string valueName, object defaultValue)
+        public static object? GetValue(string keyName, string valueName, object defaultValue)
```

Comment:
Seems [`valueName`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.registry.getvalue?view=netframework-4.8#remarks) is nullable,  technically `defaultValue` can be null too